docs/source/data_formats/dpm_cal.rst

```diff
@@ -0,0 +1,434 @@
+.. _dpm-label:
+
+Dpm Data Product
+========================================
+
+
+FITS HDU Structure
+------------------
+
+
++-------+------------------+----------+----------------------+
+| Index | Name             | Datatype | Array Size           |
++=======+==================+==========+======================+
+| 0     | Primary          | nan      | 0                    |
++-------+------------------+----------+----------------------+
+| 1     | Image            | (numpy.record, [('clocking_angle', '>f8'), ('clocking_angle_uncertainty', '>f8'), ('pos_vs_wavlen_polycoeff', '>f8', (4,)), ('pos_vs_wavlen_cov', '>f8', (4, 4)), ('wavlen_vs_pos_polycoeff', '>f8', (4,)), ('wavlen_vs_pos_cov', '>f8', (4, 4))]) | (1,)                 |
```

Comment:
This is going to break the docs page. The fix for it is kind of clunky, so this is just a reminder for me to make an issue to fix this after we pull in this PR.